mantidproject/mantid
# Elemental Analysis - Linking

**Description of work.**
Pulls together the Elemental Analysis MVP Modules in (an admittedly) messy `Elemental_Analysis.py` - this will be addressed in a future maintenance issue (#23498).

**Report to:** Aidy

**To test:**
The GUI should function as expected

Notes:
- Lines will update from the element selector (right click) when closed (not immediately when clicked)
- Lines will cycle colours - unfortunately, there's not much I can do about this right now



Closes #23426

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
